apps/meteor/server/startup/migrations/v291.ts

```diff
@@ -0,0 +1,29 @@
+import type { IAppStorageItem } from '@rocket.chat/apps-engine/server/storage';
+
+import { AppsModel } from '../../../app/models/server';
+import { AppRealStorage } from '../../../ee/server/apps/storage';
+import { addMigration } from '../../lib/migrations';
+
+addMigration({
+	version: 291,
+	name: "Mark all installed apps as 'migrated'",
+	async up() {
+		const appsStorage = new AppRealStorage(AppsModel);
+
+		const apps = await appsStorage.retrieveAll();
```

Comment:
I'd recommend using a bulkWrite instead of a promise array
apps/meteor/server/startup/migrations/v260.ts

Reply:
@d-gubert

Reply:
That bulkWrite is interesting, however since it's only a migration and we're not going to be dealing with an insane amount of records I think it would be "over-optmizing" in this specific case